illumina_metadataParser.pm

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/perl -w
+
+package illumina_metadataParser;
+
+use JSON;
+use strict;
+use warnings;
+
+BEGIN {
+
+    use Exporter;
+
+    our @ISA = ('Exporter');
+
+    our @EXPORT = qw (
+        &metadataParse
+   );
+}
+
+sub metadataParse(@){
+    my $outputdir_name = shift || return undef;
+    my $metadata_file = "$outputdir_name/metadata";
```

Comment:
I was aiming for simplicity for now, it can be upgraded in the future if we want custom metadata file names